src/payment/strategies/paypal-commerce/paypal-commerce-script-loader.ts

```diff
@@ -14,12 +14,15 @@ export default class PaypalCommerceScriptLoader {
         this._window = window;
     }
 
-    async loadPaypalCommerce(clientId: string, currency: string = 'USD'): Promise<PaypalCommerceSDK> {
-        if (!clientId) {
+    async loadPaypalCommerce(options: PaypalCommerceScriptOptions): Promise<PaypalCommerceSDK> {
+        if (!options || !options['client-id']) {
             throw new InvalidArgumentError();
         }
+        const params = (Object.keys(options) as Array<keyof PaypalCommerceScriptOptions>)
+            .map(key => `${key}=${options[key]}`)
```

Comment:
I'm thinking instead of making an exception and use a hyphenated key `client-id`, what if you make the key camel case and map it into a hyphenated key. That way we can keep the casing of the keys consistent.

Reply:
Okay. Thank you!